openmetadata-spec/src/main/resources/json/schema/entity/services/connections/dashboard/microStrategyConnection.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/dashboard/microStrategyConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "MicroStrategyConnection",
+  "description": "MicroStrategy Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.services.connections.dashboard.MicroStrategyConnection",
+  "definitions": {
+    "microStrategyType": {
+      "description": "MicroStrategy service type",
+      "type": "string",
+      "enum": ["MicroStrategy"],
+      "default": "MicroStrategy"
+    }
+  },
+  "properties": {
+    "type": {
+      "title": "Service Type",
+      "description": "Service Type",
+      "$ref": "#/definitions/microStrategyType",
+      "default": "MicroStrategy"
+    },
+    "username": {
+      "title": "Username",
+      "description": "Username to connect to MicroStrategy. This user should have privileges to read all the metadata in MicroStrategy.",
+      "type": "string"
+    },
+    "password": {
+      "title": "Password",
+      "description": "Password to connect to MicroStrategy.",
+      "type": "string",
+      "format": "password"
+    },
+    "hostPort": {
+      "expose": true,
+      "title": "Host and Port",
+      "description": "Host and Port of the MicroStrategy instance.",
+      "type": "string",
+      "format": "uri"
+    },
+    "projectName": {
+      "title": "Project Name",
+      "description": "MicroStrategy Project Name",
+      "type": "string"
+    },
+    "loginMode": {
+      "title": "Login Mode",
+      "description": "Login Mode for Microstrategy's REST API connection. You can authenticate with one of the following authentication modes: `Standard (1)`, `Anonymous (8)`. Default will be `Anonymous (8)`. If you're using demo account for Microstrategy, it will be needed to authenticate through loginMode `8`.",
+      "type": "string",
+      "default": "8"
```

Comment:
should the default be `1` then? I guess that's what an end user would need right

Reply:
Yes, mode 1 is for normal authentication. but the thing is we're not able to test it since we have demo account. Should I make `1` as default ?

Reply:
I think so making it as `1` would be appropiate as end user would be using that as per what they have in the docs [here ](https://microstrategy.github.io/rest-api-docs/common-workflows/analytics/retrieve-data-from-the-intelligence-server/workflow-retrieve-data-from-the-intelligence-server/#log-in)